frontend/src/webpack_old_index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
```

Comment:
This file will be deleted once react-router is fully integrated; it's there to serve as a model for the types of props we're passing in.